src/components/Common/Navbar/index.tsx

```diff
@@ -57,20 +58,7 @@ function Navbar() {
           Contact Us
         </button>
       </div>
-      <div
-        className="menu"
-        onClick={() => setIsOpen(!isOpen)}
-        onKeyDown={(e) => {
-          if (e.key === 'Enter' || e.key === 'Space') {
-            setIsOpen(!isOpen);
-            e.preventDefault();
-          }
-        }}
-        role="button"
-        tabIndex={0}
-      >
-        <span className="font-xlarge">Menu</span>
-      </div>
+      <MenuOutlined className="menu gray" onClick={() => setIsOpen(!isOpen)} />
```

Comment:
Possible to add a test that the Menu Button works? Maybe it can Click and go to Cultures Page from the Home Page should be fine!

Can download Playwright Testing Extension on VS Code and use Pick Locator or Record to see what the commands are like!

Reply:
Added sir